react/src/context/PostHogProvider.tsx

```diff
@@ -108,20 +90,38 @@ export function PostHogProvider({ children, client, apiKey, options }: WithOptio
             posthogJs.init(apiKey, options)
```

Comment:
Inside the `useEffect` you must call `posthog` rather `posthogJs` because we wanna initialize *that* client. I think this is working right now because we always return `posthogJs` from inside of it, but it makes more sense semantically to init that instance, right?